quattor/cluster_base.pan

```diff
@@ -0,0 +1,7 @@
+unique template quattor/cluster_base;
```

Comment:
prefixes typically indicate need for subtreem, in this case `quattor/cluster/base`

Reply:
So what do you suggest instead?

Reply:
`quattor/cluster/base`